web/src/upload_widget.ts

```diff
@@ -135,65 +344,139 @@ export function build_direct_upload_widget(
     // jQuery button to open file dialog
     $upload_button: JQuery,
     upload_function: UploadFunction,
+    // default value of max uploaded file size
     max_file_upload_size: number,
+    // profile picture or organization logo or organization icon
+    picture_type: string,
```

Comment:
This parameter mechanism doesn't work with i18n; you're passing in a fixed English string and then inserting it as a variable into a translated string, which will produce content with English mixed in.

Probably you just want a `placeholder_text` parameter that's a translated string of the form "New organization logo", and then maybe to also have the caller pass in the cropper options directly, perhaps using an exported TypeScript type.